### PR TITLE
added guide for processing transactions

### PR DESCRIPTION
**Pull request by thisisjoshford**
- added instructions & snippet for signing transactions
- added link to `transaction.ts` source code
- minor text edits

### CODE REVIEW

File: docs/roles/integrator/exchange-integration.md [hunk 1]
```diff
@@ -26,6 +26,13 @@ sidebar_label: Exchange Integration
   
   const transaction = nearAPI.transactions.createTransaction(signerId, signerPublicKey, receiverId, nonceForPublicKey, actions, blockHash);
   ```
+  - Once your transaction is constructed, you will then need to sign it by calling the `signTransactionObject` method and pass `transaction`, `signerId`, and a `networkId` (i.e. `testnet`, `betanet`, or `mainnet`)
```
Comment (bowenwang1996) [inline docs/roles/integrator/exchange-integration.md:29]: the network id is purely internal to near-api-js and is used to select the signer. We should not talk about it as if it is some general pattern

Comment (thisisjoshford) [reply docs/roles/integrator/exchange-integration.md:29]: would just removing the (i.e. `testnet`, `betanet`...) suffice, or would you rather no mention of arguments passed?

Comment (bowenwang1996) [reply docs/roles/integrator/exchange-integration.md:29]: I think we should just remove the mention of `networkId` or mention that it is specific to near-api-js

Comment (thisisjoshford) [reply docs/roles/integrator/exchange-integration.md:29]: updated @bowenwang1996 :)

**Review by bowenwang1996 — APPROVED** [review]

**Pull request merged**